inc/classes/admin/settings/class-page.php

```diff
@@ -888,21 +888,46 @@ private function media_section() {
 			]
 		);
 
-		$rocket_maybe_disable_lazyload_plugins = [];
+		$disable_images_lazyload  = [];
+		$disable_iframes_lazyload = [];
+
+		if ( rocket_avada_maybe_disable_lazyload() ) {
+			$disable_images_lazyload[] = __( 'Avada', 'rocket' );
+		}
+
 		if ( rocket_maybe_disable_lazyload() ) {
-			$rocket_maybe_disable_lazyload_plugins[] = __( 'Autoptimize', 'rocket' );
+			$disable_images_lazyload[] = __( 'Autoptimize', 'rocket' );
 		}
 
 		/**
-		 * Lazyload Helper filter which disables WPR lazyload functionality
+		 * Lazyload Helper filter which disables WPR lazyload functionality for images.
 		 *
 		 * @since  3.4.2
 		 * @author Soponar Cristina
 		 *
-		 * @param array Will return the array with all plugin names which should disable LazyLoad
+		 * @param array $disable_images_lazyload Will return the array with all plugin names which should disable LazyLoad
+		 */
+		$disable_images_lazyload = apply_filters( 'rocket_maybe_disable_lazyload_helper', $disable_images_lazyload );
+		$disable_images_lazyload = array_unique( array_filter( (array) $disable_images_lazyload ) );
+		$disable_images_lazyload = array_map( 'sprintf', array_fill( 0, count( $disable_images_lazyload ), '<strong>%s</strong>' ), $disable_images_lazyload );
+
+		/**
+		 * Lazyload Helper filter which disables WPR lazyload functionality for iframes.
+		 *
+		 * @since 3.5.5
+		 *
+		 * @param array $disable_iframes_lazyload Will return the array with all plugin names which should disable LazyLoad
 		 */
-		$rocket_maybe_disable_lazyload_plugins = apply_filters( 'rocket_maybe_disable_lazyload_helper', $rocket_maybe_disable_lazyload_plugins );
-		$rocket_maybe_disable_lazyload_plugins = wp_sprintf_l( '%l', $rocket_maybe_disable_lazyload_plugins );
+		$disable_iframes_lazyload = apply_filters( 'rocket_maybe_disable_iframes_lazyload_helper', $disable_iframes_lazyload );
+		$disable_iframes_lazyload = array_unique( array_filter( (array) $disable_iframes_lazyload ) );
+		$disable_iframes_lazyload = array_map( 'sprintf', array_fill( 0, count( $disable_iframes_lazyload ), '<strong>%s</strong>' ), $disable_iframes_lazyload );
+
+		$disable_lazyload = array_merge( $disable_images_lazyload, $disable_iframes_lazyload );
+		$disable_lazyload = array_unique( $disable_lazyload );
```

Comment:
There are a few considerations in this code that we should discuss:

1. What if the filters return an empty value? Then `array_filter` and `array_unique` are not needed. As these are expensive array functions, skipping them when there's no value is more performant.
2. The design pattern repeats. Move the `array_filter`, `array_unique`, `array_fill`, and `array_map` to a private method and return the value for each.